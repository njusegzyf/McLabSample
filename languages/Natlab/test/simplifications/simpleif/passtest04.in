if test1
    disp('then part')
elseif test2
    disp('then part2')
else
    disp('else part')
end