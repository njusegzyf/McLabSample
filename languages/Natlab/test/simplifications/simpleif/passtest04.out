if test1
    disp('then part')
else
    if test2
        disp('then part2')
    else
        disp('else part')
    end
end